CHANGELOG.md

```diff
@@ -0,0 +1,140 @@
+
+## Release Notes - Identity Service
+
+### Version: <em>identity-service-1.0</em>
+
+<h2>Epic
+</h2>
+<ul>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-134'>AUTH-134</a>] -         Identity Service 1.0
+</li>
+</ul>
+
+<h2>Story
+</h2>
+<ul>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-65'>AUTH-65</a>] -         Integrate code into APS to implement the use of the internal auth token
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-130'>AUTH-130</a>] -         Make Alfresco-Identity-Service Github repo public
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-148'>AUTH-148</a>] -         Custom Alfresco Login Theme
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-149'>AUTH-149</a>] -         Identify All Artifacts that need to be localized
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-150'>AUTH-150</a>] -         Document instructions for configuring LDAP sync
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-151'>AUTH-151</a>] -         Test LDAP sync
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-153'>AUTH-153</a>] -         Document instructions for configuring SAML
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-154'>AUTH-154</a>] -         Compliance with Alfresco Security Policy
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-155'>AUTH-155</a>] -         Complete and Update Documentation for Deploying the Identity Service
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-158'>AUTH-158</a>] -         Publish Artifacts
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-159'>AUTH-159</a>] -         Naming
```

Comment:
IMHO this discussion doesn't need to be seen externally.

Reply:
Good point.  I removed AUTH-159.